src/class_singleCleaner.py

```diff
@@ -34,6 +17,23 @@
 
 """
 
+import gc
+import threading
+import shared
+import time
+import os
+
+import tr#anslate
+from bmconfigparser import BMConfigParser
+from helper_sql import *
+from helper_threading import *
+from inventory import Inventory
+from network.connectionpool import BMConnectionPool
+from debug import logger
```

Comment:
imports should be at the top first.
later from statements should be written to the file

Reply:
Autopep8 (and hopefully flake8 or similar) is in the pipeline (#1113) which will fix this and many, many other style and lint errors.

Reply:
Extra caution required in importing project modules as there is some crappy initialisation which requires importing in a particular order. At the moment I don't recommend reordering project module imports as things may break strangely (system module import reordering is OK). This import order fix is a wholly separate project that would require some refactoring as well.